src/main/scala/net/snowflake/spark/snowflake/Parameters.scala

```diff
@@ -18,6 +18,7 @@
 package net.snowflake.spark.snowflake
 
 import com.amazonaws.auth.{AWSCredentials, BasicSessionCredentials}
+import com.microsoft.azure.storage.StorageCredentialsSharedAccessSignature
```

Comment:
I have quite a bit of concern here that we're introducing a dependency on Azure here...